TF_Tooling_Web/CHANGELOG.md

```diff
@@ -0,0 +1,63 @@
+# Changelog
+All notable changes to the TouchFree Web Tooling project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+[docs-website]: https://docs.ultraleap.com/touchfree-user-manual/ "Ultraleap Docs"
```

Comment:
👀 This line doesn't appear when viewing the page

Reply:
![image](https://user-images.githubusercontent.com/107030829/212857312-1f243184-0b26-4642-8413-33ca452f9eb3.png)

Reply:
Replaced with some badges at the top (with discord and support email too)